src/App.js

```diff
@@ -1,23 +1,132 @@
-import logo from './logo.svg';
-import './App.css';
+import "./styles/App.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+import CitySelectionForm from "./components/CitySelectionForm";
+import DisplayInformation from "./components/DisplayInformation";
+import DisplayPhotos from "./components/DisplayPhoto";
 
 function App() {
+  //Hardcoded to test Display
+  // const cityInfo = {
+  //   name: "Toronto",
+  //   country: "Canada",
+  //   population: 200000,
+  //   latitude: 51.05,
+  //   longitude: -114.067,
+  //   is_capital: false,
+  // };
+
+  //States
+  // const [city, setCity] = useState("Rio de Janeiro");
+  const [city, setCity] = useState("");
+  const [selectedInfo, setSelectedInfo] = useState({
+    // name: "Toront",
+    // country: "Canada1",
+    // population: 200000,
+    // latitude: 51.05,
+    // longitude: -114.067,
+    // is_capital: false,
```

Comment:
Not sure if this is still being used, but if not then the commented code can be removed